qaframework-bdd-tests/src/test/resources/features/refapp-2.x/07-visit/visits.feature

```diff
@@ -49,3 +49,32 @@ Feature: Patient Visits management
     And user clicks on the recent visits link in the recent visits section
     And user clicks on the end visit button
     Then the system ends the visit
+
+  @selenium
+  @patientVisit
+  Scenario: Search Active Visit
+    #User Story: Search by patient Identifier
+    Given user initiates login with patient visits
```

Comment:
`Given` key word should not be used in scenarios once it has been used in the background.

Reply:
> @Irenyak1 qa seems to be un stable at the moment.

@sherrif10  ok I will wait. Thanks for the information